tasks/generate-info.js

```diff
@@ -92,6 +99,11 @@ function getNewer(date, newer, callback) {
     callback(new Error('Trouble walking ' + sourceDir));
   });
   walker.on('end', function() {
+  
+    if (isWindows) {
+        paths = [sourceDir].concat(externsPaths);
+    }
+  
```

Comment:
The change stops this error

node tasks/build.js config/ol.json build/ol.js
ERR! Trouble generating info: The command line is too long.